chaos_genius/alerts/anomaly_alerts.py

```diff
@@ -375,7 +375,7 @@ def send_alert_email(self, anomaly_data):
                 )
                 format_anomaly_points(points)
                 self.format_alert_data(points)
-                save_anomaly_point_formatting(points)
+                save_anomaly_point_formatting(points, getattr(kpi_obj, "anomaly_params").get("frequency"))
```

Comment:
The frequency in `anomaly_params` is the time-series/data frequency. Model frequency is in `scheduler_params` -> `scheduler_frequency`.

Also, `getattr` does not provide any benefit here.

```suggestion
                save_anomaly_point_formatting(points, kpi_obj.scheduler_params.get("scheduler_frequency"))
```